Instruments/config/cross_section.cfg

```diff
@@ -59,3 +59,15 @@ WminusHToTauTau_M125: xs=0.03342 ref="Higgs_XSBR_YR4_update" unc="unknown"
 
 VBFHToTauTau_M125: xs=0.237 ref="Higgs_XSBR_YR4_update" unc="unknown"
 GluGluHToTauTau_M125: xs=3.0469  ref="Higgs_XSBR_YR4_update" unc="unknown"
+
+ggHH_SM: xs=0.02773 ref="https://indico.cern.ch/event/885347/contributions/3814736/attachments/2027454/3392224/Talk_HH_subgroup.pdf" unc="+13.8-12.8"
+
+GluGluToHHTo2B2Tau_node_cHHH0: xs=0.07038 unc="+2.4-6.1" ref="https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHXSWGHH#Latest_recommendations_for_gluon"
+GluGluToHHTo2B2Tau_node_cHHH1: xs=0.03105 unc="++2.2-5.0" ref="https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHXSWGHH#Latest_recommendations_for_gluon"
+GluGluToHHTo2B2Tau_node_cHHH2p45: xs=0.01310 unc="+2.3-5.1" ref="https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHXSWGHH#Latest_recommendations_for_gluon"
```

Comment:
xs=0.01312 * 2 * 5.824e-01 * 6.272e-02
need to be calculated for kl=2.45 using the formula from here (slide 12): https://indico.cern.ch/event/904966/contributions/3832774/attachments/2023843/3384862/HH_combine_model_21Apr2018.pdf